expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java

```diff
@@ -102,8 +103,13 @@ public ExpandedOrganization getOrganization(Entity dataEntry) throws NotFoundExc
                                       : resourceService.getResourceByIdentifier(ticketEntry.getResourceIdentifier())
                                           .getResourceOwner().getOwnerAffiliation();
 
+            var organizationIdentifier = Optional.ofNullable(organizationId)
```

Comment:
Can `organizationId` be `null`?
If no, maybe rewrite:
`var organizationIdentifier = UriWrapper.fromUri(organizationId).getLastPathElement();`

Reply:
There exists a test that specifically tests with null value here (and it fails when not checking for null-value):
[shouldUseResourceOwnerAffiliationWhenTicketHasNoOwnerAffiliation(...)](https://github.com/BIBSYSDEV/nva-publication-api/blob/main/expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java#L302)